src/frontend/apps/drive/package.json

```diff
@@ -13,7 +13,7 @@
     "node": ">=22.0.0 <23.0.0"
   },
   "dependencies": {
-    "@gouvfr-lasuite/ui-kit": "0.7.0",
+    "@gouvfr-lasuite/ui-kit": "0.8.0",
```

Comment:
Move available from the sidebar on workspaces and folders and crashes:

https://www.loom.com/share/7f217147b18840389f6367a126a15a37